static/js/jira-select-github-version.js

```diff
@@ -0,0 +1,10 @@
+$('.jiraSelectGitHubVersion__options__card').click(function (event) {
+	event.preventDefault();
+	const selectedCard = event.target.className;
+	const selectedVersion = selectedCard.includes("cloud") ? "cloud" : "server";
```

Comment:
`$(event.target).hasClass("cloud")` would be better IMO

Reply:
actually, another way of doing it would be to use `data-` attributes on the HTML element.  So in your html, you'd do something like `<div data-type="cloud">...` and for your code you just need to do `const selectedVersion = $(event.target).data("type");`